### PR TITLE
Improve the Templates regarding the Exercises

### PR DESCRIPTION
**Pull request by LucRome**
+ fix some permissions
+ some other improvements

### CODE REVIEW

**Comment by codingHahn** [comment]
This PR shares many changes with #97. Could you clarify what the difference is here?

**Comment by LucRome** [comment]
#97 contains all the foundations for this (and the other Frontend) PRs
Let's talk tomorrow about how we could show them more clearly

File: eCourse_backend/courses/urls.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
     path('iframes/studentlist/<id>', course_student_list_iframe,
          name='course_studentlist_iframe'),
     # admin: create course
-    path('create/', create_course, name="create_course"),
+    path('create/', create_course_admin, name="create_course"),
```
Comment (phie73) [inline eCourse_backend/courses/urls.py]: please revert renaming (i think it is the same as in the pr lecturer-and-student-site-2)

File: eCourse_backend/courses/views.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def detailed_course(request, id):
 
 @login_required
 @permission_required('courses.add_course', raise_exception=True)
-def create_course(request):
+def create_course_admin(request):
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: see comment in urls.py

**Review by phie73 — COMMENTED** [review]
i think you solved the comments already in the other pr?

**Comment by LucRome** [comment]
Oh sorry, forgot to merge

**Comment by LucRome** [comment]
Now the fixes should be applied...I think I made an Error when resolving the conflicts

Thanks for checking 👍

**Review by phie73 — APPROVED** [review]
looks good

**Pull request merged**